packages/mobile/android/app/libtor/include/tor/org_torproject_jni_TorService.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright (c) 2019, Matthew Finkel.
+  * Copyright (c) 2019, Hans-Christoph Steiner.
+  * Copyright (c) 2007-2019, The Tor Project, Inc. */
+ /* See LICENSE for licensing information */
```

Comment:
I'm not exactly sure the best way to handle this, but we might want to improve our attribution so that folks know where this code is coming from.

Reply:
If I understand correctly we need to add their license content to our repo:
https://github.com/guardianproject/tor-android/tree/master?tab=License-1-ov-file
![image](https://github.com/TryQuiet/quiet/assets/6099829/85ad16de-4001-4eca-a5de-70fc6df813ca)

Reply:
Talking with folks, for code like this, I think we should put a comment at the top that links to the project where the code is from, a link to a copy of the license in our repo and a note about whether we modified the code or not and a brief description of the modifications.

The link to the repo can be a git permalink (referencing a specific commit/file).

Copies of external licenses can live in an `external-licenses` directory. We can copy the license from guardian project or wherever and put it under `external-licenses/guardian-project.txt` or something like that.

So altogether, in this file, we might have something like this at the top:

```
/**
 * The code in this file is from <link to repo>, is licensed under external-licenses/guardian-project.txt and has been modified by us. Can talk further about modifications briefly here.
 */
 
/* Copyright (c) 2019, Matthew Finkel.
  * Copyright (c) 2019, Hans-Christoph Steiner.
  * Copyright (c) 2007-2019, The Tor Project, Inc. */
 /* See LICENSE for licensing information */
```

Of course if the file wasn't modified, we can leave out that part.

Reply:
And we can do something similar for any other code files in this PR that are from other repos.